### DISCUSSION TITLE
LATEX公式bug求问

### DISCUSSION THREAD

**Discussion by 1102liu** [windingwind/zotero-better-notes#766] [Q&A]
在使用latex代码写公式的时候出现一个问题，首次编辑$*$后显示正常，当二次对公式进行编辑，公式后方无法输入中文或英文，仅能输入数字，尝试换行之类的也不行，想问下这个是啥原因啊

例如：

正文正文正文$公式内容$ 后面可以正常输入

当对“公示内容”部分进行修改后，就没办法在后面输入文字了，只能输入数字后才能继续打字

**Comment by windingwind** [comment]
与插件无关，向zotero论坛反馈

**Reply by 1102liu** [reply]
那没事了
